compose/runtime/runtime/src/nativeMain/kotlin/androidx/compose/runtime/MonotonicFrameClock.native.kt

```diff
@@ -31,8 +31,10 @@ import kotlinx.coroutines.yield
 )
 public actual val DefaultMonotonicFrameClock: MonotonicFrameClock =
     object : MonotonicFrameClock {
+        private val markNow = TimeSource.Monotonic.markNow()
```

Comment:
This module is fully upstreamed, and this code is not used after #2221
Let's do it in AOSP only and get the change during the regular merge

Reply:
right.

Reply:
fixed